packages/forms-web-app/src/services/user.service.js

```diff
@@ -139,8 +139,7 @@ const setLPAUserStatus = async (req, userId, status) => {
  */
 
 const isRule6UserByEmail = async (req, email) => {
-	const isRule6User = await req.appealsApiClient.isRule6User(email);
-	return isRule6User;
+	return await req.appealsApiClient.isRule6User(email);
```

Comment:
can lose the await here

Reply:
Removed, unfortunately that dismisses your review